Gemfile.lock

```diff
@@ -59,13 +59,13 @@ GEM
     autoprefixer-rails (4.0.2.2)
       execjs
     awesome_print (1.2.0)
-    aws-sdk (1.59.1)
-      aws-sdk-v1 (= 1.59.1)
-    aws-sdk-v1 (1.59.1)
+    aws-sdk (1.60.2)
```

Comment:
We use AWS storage for Paperclip, so it belongs here. My question is about these gems being updated without (most likely) being dependencies of either `will_paginate` or `mocha` - did you run `bundle update` at some point?

If these updates are not causing any failures, it's not a bad thing to let them happen, but as a general practice gem updating should be confined to its own PRs.